metrics/measured_http/http.go

```diff
@@ -55,17 +58,34 @@ func New(m serveMux, clk clock.Clock, stats prometheus.Registerer, opts ...otelh
 		},
 		[]string{"endpoint", "method", "code"})
 	stats.MustRegister(responseTime)
+
+	inFlightRequestsGauge := prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "in_flight_requests",
+			Help: "Tracks the number of WFE requests currently in flight, labeled by endpoint.",
+		},
+		[]string{"method"},
```

Comment:
I think this is meant to be endpoint:

```suggestion
		[]string{"endpoint"},
```